tests/system.py

```diff
@@ -115,8 +115,12 @@ def setUpModule():
     Config.IN_EMULATOR = os.getenv(BIGTABLE_EMULATOR) is not None
 
     if Config.IN_EMULATOR:
-        credentials = EmulatorCreds()
-        Config.CLIENT = Client(admin=True, credentials=credentials)
+        # I expect users won't always pass creds, just use the usual cred flow
```

Comment:
This could be removed I think. I am not really familiar with why we were doing this, but perhaps it is worth digging into?